### ISSUE TITLE
No such file as test.txt?

### ISSUE THREAD

**Issue by Timonzimm** [yhenon/keras-frcnn#4]
When trying to launch with PascalVOC 2007 or 2012 I get the following error:

No such file or directory: 'data/VOCtrainval_06-Nov-2007/VOCdevkit/VOC2007/ImageSets/Main/test.txt'

Is this file supposed to be in the same archive or downloaded separately, I must be missing something?

**Comment by yhenon** [comment]
Hi,
Thanks for your report. I'm looking into this - my local copy of pascal VOC does include the `test.txt` for VOC2007, but not for VOC2012. 

However, the exception should be caught and the training should continue. Note that this is to ensure we don't accidentally train on the test set - it is not a critical error.

If it makes things clearer, I can add some handling for not finding the `test.txt` file.

**Comment by Timonzimm** [comment]
Thank you for your quick reply! I would definitely add something to handle this case.
